schema/url/pass/plugin-pieplot.raml

```diff
@@ -0,0 +1,54 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  PieplotPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: PieplotSpec
+  
+  # were making overlay required, since this plot has no formal axes in the schema
+  PieplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
+        type: string
+      showMissingness:
+        type: string
+        enum: ['TRUE', 'FALSE']
+        required: false
+      overlayVariable: 
+        type: VariableSpec
+      facetVariable: 
+        type: VariableSpec[]
+        maxItems: 2
+        required: false
+      valueSpec: 
+        type: ValueSpec
```

Comment:
> it does seem to make intuitive sense to have the two (valueSpec and ranking method) be consistent if its not going to be explicitly user defined

I didn't quite follow - does this mean we're planning to always use the same ranking method? So no user-chosen ranking method? Totally fine with me, just making sure we don't need a `rankingMethod` property here.

Reply:
think thats exactly what were saying. or at the very least if we do choose to offer a user-defined ranking method at some point we can figure it out then